### ISSUE TITLE
Update all the badges in repos

### ISSUE THREAD

**Issue by diasdavid** [ipfs/testing#55]
With the move to Codecov and Jenkins, pretty much every repo has now their badges wrong misinforming users. Can we get a script that does an update like this -- https://github.com/ipfs/js-ipfs-unixfs/pull/31/files -- to all the repos?

**Comment by VictorBjelkholm** [comment]
Yeah, we can automate this, if the title of the links are the same in every repository, it should be a quick find-and-replace.
